hammer/reporting-remediation/reporting/create_elasticsearch_domain_logging_issue_tickets.py

```diff
@@ -0,0 +1,165 @@
+                            jira.add_label(
+                                ticket_id=issue.jira_details.ticket,
+                                label=IssueStatus.Whitelisted.value
+                            )
+                        jira.close_issue(
+                            ticket_id=issue.jira_details.ticket,
+                            comment=comment
+                        )
+                        slack.report_issue(
+                            msg=f"{comment}"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_closed(ddb_table, issue)
+                    # issue.status != IssueStatus.Closed (should be IssueStatus.Open)
+                    elif issue.timestamps.updated > issue.timestamps.reported:
+                        logging.error(f"TODO: update jira ticket with new data: {table_name}, {account_id}, {domain_name}")
+                        slack.report_issue(
+                            msg=f"Elasticsearch domain logging '{domain_name}' issue is changed "
+                                f"in '{account_name} / {account_id}' account, '{region}' region"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_updated(ddb_table, issue)
+                    else:
+                        logging.debug(f"No changes for '{domain_name}'")
+                # issue has not been reported yet
+                else:
+                    logging.debug(f"Reporting Elasticsearch domain logging '{domain_name}' issue")
+
+                    owner = tags.get("owner", None)
+                    bu = tags.get("bu", None)
+                    product = tags.get("product", None)
+
+                    issue_description = (
+                        f"Elasticsearch domain logging is not enabled.\n\n"
+                        f"*Risk*: High\n\n"
+                        f"*Account Name*: {account_name}\n"
+                        f"*Account ID*: {account_id}\n"
+                        f"*Region*: {region}\n"
+                        f"*Domain ID*: {domain_name}\n"
+                    )
+
+                    issue_description += JiraOperations.build_tags_table(tags)
+
+                    auto_remediation_date = (self.config.now + self.config.esLogging.issue_retention_date).date()
+                    issue_description += f"\n{{color:red}}*Auto-Remediation Date*: {auto_remediation_date}{{color}}\n\n"
```

Comment:
Do we need to add this comment if auto-remediation is actually disabled?

Reply:
Added condition based autoremediation.